ros_controllers/src/diff_drive_controller.cpp

```diff
@@ -14,12 +14,272 @@
 
 #include "ros_controllers/diff_drive_controller.hpp"
 
-namespace
+#include <lifecycle_msgs/msg/state.hpp>
+#include <utility>
```

Comment:
Make sure to follow the [Google C++ style guide for include headers](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).

These two should be above line#15 as:

```
#include <utility>
#include <lifecycle_msgs/msg/state.hpp>
```

Reply:
My bad. I thought I put it through clang format. I can format it with uncrustify like you ssaid.

Reply:
Yes please, otherwise CI will complain about it.